package.json

```diff
@@ -51,6 +51,7 @@
     "@stripe/stripe-js": "^1.35.0",
     "@vue-stripe/vue-stripe": "^4.4.2",
     "axios": "^0.27.2",
+    "axios-cache-interceptor": "^0.10.7",
```

Comment:
This is probably a leftover from your testing for the cache implementation, it should not be included in this PR yet, since it's unrelated 😉  You should also remove the changes to package-lock.json

Reply:
I will keep this "relatedness" priority in mind. Thanks.